qutebrowser/browser/commands.py

```diff
@@ -59,7 +59,10 @@ def _new_tabbed_browser(self, private):
 
     def _count(self) -> int:
         """Convenience method to get the widget count."""
-        return self._tabbed_browser.widget.count()
+        count = self._tabbed_browser.widget.count()
+        if count < 0:
```

Comment:
I don't think this makes sense. While the first index is `0`, the count when there are no tabs is `0`. There should never be a negative count, so this will never trigger.